Mathlib/Algebra/Group/Defs.lean

```diff
@@ -3,13 +3,14 @@ Copyright (c) 2014 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad, Leonardo de Moura, Simon Hudon, Mario Carneiro
 -/
-import Mathlib.Algebra.Notation.Defs
+import Batteries.Logic
+import Mathlib.Algebra.Regular.Defs
 import Mathlib.Data.Int.Notation
 import Mathlib.Data.Nat.BinaryRec
 import Mathlib.Logic.Function.Defs
-import Mathlib.Tactic.Simps.Basic
+import Mathlib.Tactic.MkIffOfInductiveProp
```

Comment:
The increased imports are due to this new import (for `@[mk_iff]` attribute). I hope this is okay, as more files have imports reduced.